test/unit/org/apache/cassandra/service/ActiveRepairServiceTest.java

```diff
@@ -90,6 +87,7 @@ public void prepare() throws Exception
 
         TokenMetadata tmd = StorageService.instance.getTokenMetadata();
         tmd.clearUnsafe();
+        SystemKeyspace.setBootstrapState(SystemKeyspace.BootstrapState.COMPLETED);
```

Comment:
Why is this change needed for this test?

Reply:
I believe the test initialization requires it here and the other test suite w/ the new assertions

Reply:
Because `setTokens` calls `setGossipTokens`, got it